File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandservice",function ($http) {
    this.add=function () {
        return $http.get("../brand/findAll.do");
    };
    this.get=function (id) {
        return $http.get("../brand/findById.do?id=" + id)
    };
    this.save=function (entity) {
        return $http.post("../brand/save.do",entity)
    };
    this.delet=function (entity) {
        return $http.get("../brand/delete.do?ids="+entity);
    };
    this.search=function (page,size,entity) {
        return $http.post("../brand/search.do?page=" + page + "&size=" + size,entity)
    }
});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.entity.PageResult;
import com.pinyougou.pojo.TbBrand;

import java.util.List;

public interface BrandService {
    /**
     * 返回全部列表
     *
     * @return
     * @param pageNum
     * @param pageSize
     */
    PageResult findAll(Integer pageNum, Integer pageSize);

    Boolean update(TbBrand tbBrand);

    Boolean insert(TbBrand tbBrand);

    TbBrand findById(Long id);

    void delete(Long id);

    PageResult search(TbBrand tbBrand, Integer page, Integer size);
}
